middleman_protocol/middleman_protocol/exceptions.py

```diff
@@ -24,3 +24,7 @@ class FrameInvalidMiddlemanProtocolError(MiddlemanProtocolError):
 
 class BrokenEscapingInFrameMiddlemanProtocolError(MiddlemanProtocolError):
     pass
+
+
+class PayloadLengthMismatchMiddlemanProtocolError(MiddlemanProtocolError):
```

Comment:
`PayloadLength` is no longer a field, it can be removed.

Reply:
Done